### DISCUSSION TITLE
Error: Kubernetes cluster unreachable: Get "http://<ip-private>:6443/version?":  dial tcp <ip-private>:6443:  i/o timeout

### DISCUSSION THREAD

**Discussion by crisda100** [oracle-quickstart/oci-kubernetes-monitoring#70] [Q&A]
**Error: Kubernetes cluster unreachable: Get "http://<ip-private>:6443/version?":  dial tcp <ip-private>:6443:  i/o timeout**
Hello,

when deploying this solution in a private cluster through resource manager in oci. I get this error that doesn't let me complete the deployment.

help thanks

**Comment by paliwalparitosh** [comment]
The helm chart installation through Resource Manager in private OKE clusters is not yet available.

In upcoming release (~2-3 weeks), you would be able to choose an "OCI Resource Only" Installation which shall output the required helm commands to complete the monitoring setup.
Those helm commands should be executed from a node where cluster is accessible from.

At present, you can use the RMS Stack for monitoring public OKE clusters or opt for [manual](https://github.com/oracle-quickstart/oci-kubernetes-monitoring?tab=readme-ov-file#helm) (recommended for production) helm installation method.
